drill-cicd-gradle-plugin/src/main/kotlin/com/epam/drill/integration/gradle/AgentRunnerTasks.kt

```diff
@@ -59,6 +61,12 @@ fun modifyToRunDrillAgents(
                         mapGeneralAgentProperties(it, pluginConfig.appAgent, pluginConfig)
                         this.appId = pluginConfig.appId
                         this.packagePrefixes = pluginConfig.packagePrefixes
+                        this.buildVersion = pluginConfig.buildVersion
+                        this.commitSha = runCatching {
+                            gitClient.getCurrentCommitSha()
+                        }.onFailure {
+                            logger.warn("Cannot get current commit sha, commitSha parameter will be skipped: ${it.message}")
```

Comment:
`"Unable to retrieve the current commit SHA. The 'commitSha' parameter will not be set. Error: ${it.message}"`
(according to chatgpt "unable to retrieve" sound more formal / polished)